### ISSUE TITLE
I absolutely love this!!

### ISSUE THREAD

**Issue by cedric-cle** [romboHQ/tailwindcss-motion#9]
The editor has HUUUGE potential man!

I was wondering if this could be enhanced to be not only tailwind-ish.
Like agnostic or "vanilla" scss/css compatible (with the editor).

I have been able to reverse engineer it via play.tailwind but that's quite inconvenient due to the need of copying in the playground where the motion css variables and classes are compiled in vanilla css.

Do you think the editor and the general animation mechanic could be enhanced?

Kind regards :)

**Comment by KevinGrajeda** [comment]
Yep, we are planning for it to output other type of syntax(vanilla CSS, Framer Motion, GSAP), but first we are working on getting the builder in parity with more Tailwind Motion functionalities. You can sign up for updates https://rombo.co/tailwind/#waitlist or follow [Rombo](https://x.com/romboHQ) or [me](https://x.com/k_grajeda) on Twitter.

**Comment by KevinGrajeda** [comment]
@KeoChanmaridoDev can you share a minimal reproduction of your issue?

**Comment by KeoChanmaridoDev** [comment]
Yes, I can.
Sorry for future inconvenience
As the image below shows me this.
[image: image.png]

but the class motion when I try to call it show, but it does not work.
i try many times to delete it and try again but still no luck

**Comment by ianeblack** [comment]
i do

```
bun add -D tailwindcss-motion


and then i add it to the tailwind.config.ts but i get the error bellow 
```

<img width="2632" alt="Screenshot 2024-12-24 at 11 03 01" src="https://github.com/user-attachments/assets/130de186-d01d-4d3e-97a8-6cf621681cec" />

**Comment by ianeblack** [comment]
> @KevinGrajeda
> 
> i do
> 
> `bun add -D tailwindcss-motion`
> 
> and then i add it to the tailwind.config.ts but i get the error bellow
> 
> <img alt="Screenshot 2024-12-24 at 11 03 01" width="2000" src="https://private-user-images.githubusercontent.com/117324200/398435198-130de186-d01d-4d3e-97a8-6cf621681cec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwNTM4NDIsIm5iZiI6MTczNTA1MzU0MiwicGF0aCI6Ii8xMTczMjQyMDAvMzk4NDM1MTk4LTEzMGRlMTg2LWQwMWQtNGQzZS05N2E4LTZjZjYyMTY4MWNlYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyNFQxNTE5MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMzM4N2RiOGQ1YTIwYjFiNzViMTBlMWUzM2FkNGNiMWIyNzAxYzk3OGZjMDVmNDg4MWZhYjZjMWU5N2ExYTg0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Zfjddze0CsEEfbdEPxYYJXfCM3ScUG0n-vZpJBFSYsg">

Ok so i manage to fix it by making a `tailwindcss-motion.d.ts` in my `/src` path and adding this code:

```
declare module "tailwindcss-motion";
```

Hope this helps yall out.

**Comment by KevinGrajeda** [comment]
this should be fixed with v1.0.1 🫡

**Comment by franklinjavier** [comment]
> this should be fixed with v1.0.1 🫡

I'm using 1.0.1 and the error persists.

```sh
tailwind.config.ts:4:8 - error TS1192: Module '"/Users/frank/www/node_modules/.pnpm/tailwindcss-motion@1.0.1_tailwindcss@3.4.15_ts-node@10.9.2_@types+node@22.10.5_typescript@5.7.2__/node_modules/tailwindcss-motion/src/index"' has no default export.

4 import tailwindcssMotion from 'tailwindcss-motion'
         ~~~~~~~~~~~~~~~~~

Found 1 error in tailwind.config.ts:4

```

**Comment by franklinjavier** [comment]
Fixed with 

```ts
import * as tailwindcssMotion from 'tailwindcss-motion'
```
